src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
I added exception in case of failed open. I think fail to open project is a situation that should be explicitly resolved by script writer because if `open_project` will just return false it can be ignored and all following actions will be performed on wrong project. With exception approach I think bool status is useless here